homework.md

```diff
@@ -91,14 +93,13 @@ A place for us to store and share our code.
 ## Question 15
 
 What does it mean to "Commit your changes"?
-
-Send you code to Github
+It means that I am sure to the changes I made.
```

Comment:
When we commit changes we are storing the changes that we have made locally on our machine to our branch of the repository